Source/Letterbook.Core/Models/Mappers/ProfileMappings.cs

```diff
@@ -9,16 +12,40 @@ public class ProfileMappings : AutoMapper.Profile
 	public ProfileMappings()
 	{
 		CreateMap<Models.Profile, FullProfileDto>(MemberList.Destination)
+			.ForMember(dto => dto.Followers, opt => opt.MapFrom(src => src.FollowersCount))
+			.ForMember(dto => dto.Following, opt => opt.MapFrom(src => src.FollowingCount))
 			.ForMember(dto => dto.Created, opt => opt.Ignore());
 		CreateMap<FullProfileDto, Models.Profile>(MemberList.Source)
 			.ForSourceMember(dto => dto.Updated, opt => opt.DoNotValidate())
 			.ForSourceMember(dto => dto.Created, opt => opt.DoNotValidate())
 			.ForSourceMember(dto => dto.Keys, opt => opt.DoNotValidate())
+			.ForSourceMember(dto => dto.Followers, opt => opt.DoNotValidate())
+			.ForSourceMember(dto => dto.Following, opt => opt.DoNotValidate())
+			.ForMember(profile => profile.Followers, opt => opt.Ignore())
+			.ForMember(profile => profile.Following, opt => opt.Ignore())
 			.ForMember(profile => profile.Updated, opt => opt.Ignore())
 			.ForMember(profile => profile.Keys, opt => opt.Ignore());
 		CreateMap<Models.Profile, MiniProfileDto>(MemberList.Destination);
 
-		CreateMap<Models.SigningKey, PublicKeyDto>()
-			.ForMember(dto => dto.PublicKeyPem, opt => opt.MapFrom<PublicKeyPemConverter>());
+		CreateMap<SigningKey, PublicKeyDto>()
+			.ConvertUsing(src => new PublicKeyDto
+			{
+				Label = src.Label,
+				Family = src.Family.ToString(),
+				PublicKeyPem = PemStringBuilder(src.PublicKey).ToString().Trim(),
+				Created = src.Created,
+				Expires = src.Expires,
+				FediId = src.FediId
+			});
+	}
+
+	private static StringBuilder PemStringBuilder(ReadOnlyMemory<byte> publicKey)
+	{
+		var builder = new StringBuilder();
+		using TextWriter writer = new StringWriter(builder);
+		var pemWriter = new PemWriter(writer);
+		var pem = new PemObject("PUBLIC KEY", publicKey.ToArray());
+		pemWriter.WriteObject(pem);
+		return builder;
```

Comment:
TBH, this is a nicer implementation, anyway. And, this method reduces our reliance on the System.Security.Cryptography implementations. If this can replace our uses of `AssymetricAlgorithm.ExportSubjectPublicKeyInfoPem()`, then we can also switch to BouncyCastle for key generation, and we'd be able to use ed25519 signatures.